src/Build/BackEnd/Components/RequestBuilder/RequestBuilder.cs

```diff
@@ -1105,11 +1106,11 @@ private async Task<BuildResult> BuildProject()
             ErrorUtilities.VerifyThrow(_targetBuilder != null, "Target builder is null");
 
             // We consider this the entrypoint for the project build for purposes of BuildCheck processing 
-            bool isRestoring = _requestEntry.RequestConfiguration.GlobalProperties[MSBuildConstants.MSBuildIsRestoring] is null;
+            bool isRestoring = _requestEntry.RequestConfiguration.GlobalProperties[MSBuildConstants.MSBuildIsRestoring] is not null;
```

Comment:
nit: Is the change required within this pr?

Reply:
not necessary - but make the code more understandable.

I spent >20 minutes debugging the code with conditional breakpoints skipping the restore phase - only to find out that `isRestoring` means 'The current phase is *not* Restore'